src/components/DayPickerRangeController.jsx

```diff
@@ -41,6 +41,7 @@ const propTypes = {
   withPortal: PropTypes.bool,
   hidden: PropTypes.bool,
   initialVisibleMonth: PropTypes.func,
+  scrollable: PropTypes.bool,
```

Comment:
The current `DateRangePicker`'s popup calendar doesn't seamlessly fit the new scrolling calendars, and my goal is to create a new component like `ScrollableDateRangePicker` that combines a fixed `DateInput` header with the scrolling region below.